server/views/components/totalAndCost.njk

```diff
@@ -1,16 +1,49 @@
-{% macro totalAndCost(field) %}
+{% macro totalAndCost(field, isFirstRow = false, isTotalRow = false) %}
     {% set totalAndCost = field.value %}
-    <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">{{ field.label }}</dt>
-        <dd class="govuk-summary-list__value">
-            <div class="govuk-grid-row">
-                <div class="govuk-grid-column-one-third">
-                    <span class="govuk-body">{{ totalAndCost.total }}</span>
+    {% if isFirstRow %}
+        <div class="govuk-summary-list__row custom-cost-header">
+            <dt class="govuk-summary-list__key"></dt>
+            <dd class="govuk-summary-list__value">
+                <div class="govuk-grid-row">
+                    <div class="govuk-grid-column-one-half">
+                        <div class="govuk-body"></div>
+                    </div>
+                    <div class="govuk-grid-column-one-half">
+                        <div><h3>Cost</h3></div>
+                    </div>
                 </div>
-                <div class="govuk-grid-column-one-third">
-                    <span class="govuk-body">{{ totalAndCost.cost | formatCurrency }}</span>
+            </dd>
+        </div>
+    {% endif %}
+    {% if isTotalRow %}
+        <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key">{{ field.label }}</dt>
+                <dd class="govuk-summary-list__value">
+
+                    <div class="govuk-grid-row">
+                        <div class="govuk-grid-column-one-half">
+                            <div class="govuk-body govuk-!-padding-top-2 govuk-!-static-margin-bottom-0"> </div>
+                        </div>
+                        <div class="govuk-grid-column-one-half">
+                            <div class="govuk-body govuk-!-padding-top-2 govuk-!-static-margin-bottom-0">{{ totalAndCost.cost | formatCurrency }}</div>
+                        </div>
+                    </div>
+                </dd>
+
```

Comment:
This `total AndCost` line doesn't have the same behaviour than the second one:
- margin on the left side
- can it stand on 1 line rather than 2?
- `cost` column is squeeze on the left side

Reply:
done